dto/decision_dto.go

```diff
@@ -98,9 +103,22 @@ func NewAPIDecision(decision models.Decision) APIDecision {
 	return apiDecision
 }
 
-func NewAPIDecisionWithRule(decision models.DecisionWithRuleExecutions) APIDecisionWithRules {
+func toDecisionUrl(marbleAppHost string, decisionId string) null.String {
+	if marbleAppHost == "" {
+		return null.String{}
+	}
+
+	url := url.URL{
+		Scheme: "https",
```

Comment:
If you want to allow specific unsafe protocol for dev, you can do a special case for local development (cf suggestion on CORS)

Reply:
Here it's more annoying, mostly because I really don't want to inject the environment variable all the way down to the DTO adapter. 
So while it works for CORS, I kinda don't want to do it here